pkg/controller/routes.go

```diff
@@ -128,7 +128,7 @@ func routes(router *router.Router, cfg *rest.Config, registryTransport http.Roun
 	router.Type(&netv1.Ingress{}).Selector(managedSelector).Namespace(system.ImagesNamespace).HandlerFunc(gc.GCOrphans)
 	router.Type(&corev1.Secret{}).Selector(managedSelector).Name(system.DNSSecretName).Namespace(system.Namespace).HandlerFunc(secrets.HandleDNSSecret)
 	router.Type(&netv1.Ingress{}).Selector(managedSelector).Name(system.DNSIngressName).Namespace(system.Namespace).Middleware(ingress.RequireLBs).Handler(ingress.NewDNSHandler())
-	router.Type(&corev1.Secret{}).Selector(managedSelector).Middleware(tls.RequireSecretTypeTLS).HandlerFunc(tls.RenewCert) // renew (expired) TLS certificates, including the oss-acorn.io wildcard cert
+	router.Type(&corev1.Secret{}).Selector(managedSelector).Namespace(system.Namespace).Middleware(tls.RequireSecretTypeTLS, tls.AcornWildcardCertOnly).HandlerFunc(tls.RenewCert) // renew (expired) TLS certificate
```

Comment:
Can we use the `Name` method here instead of the middleware?

Reply:
We absolutely should. No idea how I overlooked that.